### PR TITLE
106 fix users in singlecardfragment

### PR DESCRIPTION
**Pull request by KelvinKappeler**
If this PR is accepted, it will fix some issues with the list of users who want / have a card in SingleCardFragment.

### CODE REVIEW

File: app/src/main/java/com/github/sdp/tarjetakuna/ui/home/HomeFragment.kt [hunk 1]
```diff
@@ -35,6 +35,11 @@ class HomeFragment : Fragment() {
         _binding = FragmentHomeBinding.inflate(inflater, container, false)
         val root: View = binding.root
 
+        //LocalDatabaseProvider.deleteDatabases(
```
Comment (Bjolidon) [inline app/src/main/java/com/github/sdp/tarjetakuna/ui/home/HomeFragment.kt:38]: Don't forget to remove this

File: app/src/main/java/com/github/sdp/tarjetakuna/ui/singlecard/SingleCardFragment.kt [hunk 2]
```diff
@@ -40,6 +43,9 @@ class SingleCardFragment : Fragment() {
         _binding = FragmentSingleCardBinding.inflate(inflater, container, false)
         viewModel = ViewModelProvider(this)[SingleCardViewModel::class.java]
 
+        UserRTDB(FirebaseDB()).getUsers().thenApply {
+            Log.d("BLABLA", it.toString())
```
Comment (Bjolidon) [inline app/src/main/java/com/github/sdp/tarjetakuna/ui/singlecard/SingleCardFragment.kt:47]: What is this log?

**Review by Bjolidon — CHANGES_REQUESTED** [review]
The feature seems to work well, good job!

File: app/src/main/java/com/github/sdp/tarjetakuna/ui/singlecard/SingleCardFragment.kt (hunk 2 above)
Comment (KelvinKappeler) [reply app/src/main/java/com/github/sdp/tarjetakuna/ui/singlecard/SingleCardFragment.kt:47]: Thanks for your remark.

File: app/src/main/java/com/github/sdp/tarjetakuna/ui/home/HomeFragment.kt (hunk 1 above)
Comment (KelvinKappeler) [reply app/src/main/java/com/github/sdp/tarjetakuna/ui/home/HomeFragment.kt:38]: Thanks for your remark.

**Review by Bjolidon — DISMISSED** [review]

**Pull request closed without merging**